tests/unit/model/native-access-arrays-test.js

```diff
@@ -99,17 +99,6 @@ if (CUSTOM_MODEL_CLASS) {
       assert.equal(slicedChapters[0].get('name'), 'The Boy Who Lived', `Array.slice still works`);
       assert.notEqual(slicedChapters, chapters, `Array.slice still works`);
 
-      let shiftedChapters = chapters.slice();
-      let shiftedChapter = shiftedChapters.shift();
-
-      assert.equal(shiftedChapters[0].get('name'), 'The Boy Who Lived', `Array.shift works`);
-      assert.equal(shiftedChapters.length, 1, `Array.shift works`);
-      assert.equal(shiftedChapter.get('name'), 'The Vanishing Glass', 'Array.shift works');
-
-      shiftedChapters.unshift(shiftedChapter);
-      assert.equal(shiftedChapters[0].get('name'), 'The Vanishing Glass', `Array.unshift works`);
-      assert.equal(shiftedChapters.length, 2, `Array.unshift works`);
-
```

Comment:
why are these removed?

Reply:
These have been moved to https://github.com/hjdivad/ember-m3/pull/1291/files/ea086c142eeb5656dcd4f8135c5038fa4ffaeae7#diff-ca4b95cfbe6d1d2a7c6d1b9d8d44b4593ee6b0470174d1e49b004bfd14bb65b1R171 to simplify the tests